include/erlcloud_aws.hrl

```diff
@@ -27,6 +27,10 @@
           kinesis_host="kinesis.us-east-1.amazonaws.com"::string(),
           kinesis_port=80::non_neg_integer(),
           kinesis_retry=fun erlcloud_kinesis_impl:retry/2::erlcloud_kinesis_impl:retry_fun(),
+          kms_scheme="https://"::string(),
+          kms_host="kms.us-east-1.amazonaws.com"::string(),
+          kms_port=80::non_neg_integer(),
+          kms_retry=fun erlcloud_kms_impl:retry/2::erlcloud_kms_impl:retry_fun(),
```

Comment:
https://github.com/alertlogic/erlcloud/pull/13/files#diff-d9bda54b3e3a2f222e12d368ffd0d4ffR1009 `erlcloud_retry:retry` is already used and there is not file `erlcloud_kms_impl` so lets fix this default too.